src/Pages/UserHint/UserHint.jsx

```diff
@@ -0,0 +1,21 @@
+import { useNavigate } from 'react-router-dom'
+import { JsData } from '../../Backend/Data'
+import { Account } from '../../Components/Other/Account'
+import { LANDING_PAGE_PATH } from '../../Constants/Constants'
+
+const allCompanies = JsData.searchData.companies
+
+export const UserHint = () => {
+  const navigate = useNavigate()
+
+  return (
+    <Account
+      title='Beneath are the companies in the database'
+      onClick={() => navigate(LANDING_PAGE_PATH)}
```

Comment:
usually it's not ideal to use buttons for redirecting / navigating, check if you can use a link :)